crates/bevy_pbr/src/lib.rs

```diff
@@ -166,7 +174,7 @@ impl Plugin for PbrPlugin {
             .init_resource::<ShadowPipeline>()
             .init_resource::<DrawFunctions<Shadow>>()
             .init_resource::<LightMeta>()
-            .init_resource::<GlobalLightMeta>()
+            .insert_resource(GlobalLightMeta::new(use_storage_buffers))
```

Comment:
This style of API is unclear, because it takes a raw bool in a positional argument. As we extend it, there's a serious risk that we end up with seven different bools, whose order matters in arbitrary ways.

I would prefer either raw construction of the struct or a custom enum type.

Reply:
I decided to use a bitflags struct. Let me know what you think.

Reply:
Good, I like this struct: looks like it should be nicer to maintain and extend.